ubid.rb

```diff
@@ -190,6 +190,10 @@ def to_i
     @value
   end
 
+  def inspect
+    "#<UBID:#{TYPE2CLASS[to_s[..1]] || "Unknown"} @ubid=#{to_s.inspect} @uuid=#{to_uuid.inspect}>"
```

Comment:
do you need `.inspect` here? Feels like `to_s` should be enough?

Reply:
When we don't add `.inspect`, it prints the string without quotes. It makes hard to differentiate value and it's recommended to put all values in `.inspect` representation.

with `.inspect`: 

```ruby
[1] clover-development(main)> UBID.parse("vmy2mfzyj1yj2kv7rpevcp9afs")
=> #<UBID:Vm @ubid="vmy2mfzyj1yj2kv7rpevcp9afs" @uuid="f0a8fffa-41f4-8774-9ecf-8b3b6cb254fc">
```

without `.inspect`: 
```ruby
[1] clover-development(main)> UBID.parse("vmy2mfzyj1yj2kv7rpevcp9afs")
=> #<UBID:Vm @ubid=vmy2mfzyj1yj2kv7rpevcp9afs @uuid=f0a8fffa-41f4-8774-9ecf-8b3b6cb254fc>
```

Some recommendations:
 
> You’ll also note that we call inspect on each attribute, which ensures the value as a whole is returned as expected. If we did not do this, the name attribute would render without quotation marks. This is an issue because it makes it look like the name is a Ralph class, rather than a String.

```
Person.new("Ralph", 20)
# => #<Person @name="Ralph" @age=20>
Person.new("Ralph", 20).inspect
# => "#<Person @name=Ralph @age=20>" # <- Note the missing quotation marks on Ralph
```

Reply:
I see, alright 🚢